interface/build.config.js

```diff
@@ -78,6 +78,7 @@ module.exports = {
       'vendor/ng-sortable/dist/ng-sortable.min.js',
       'vendor/angularjs-slider/dist/rzslider.min.js',
       'vendor/d3-tip/index.js',
+      'd3.network.js',
```

Comment:
:+1: It's great that we are now including this in the code for adage-server. However, it does mean that we need to manually update the `interface/d3.network.js` file everytime we make a change to the code in the repository. Ideally this would be an npm installable package or something of the sort. We might need to make an issue for this - what do you think?

Reply:
I think we can create a `d3.network` repo in GreeneLab, and I will fork it for code review, and use npm to install the one in GreeneLab. If this is okay with you guys, I will create an issue.

Reply:
:+1: